site/website/crafter-level-descriptor.level.xml

```diff
@@ -15,17 +15,12 @@
   ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
   -->
 
-<levelDescriptor
-        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-
-    <header xsi:nil="false">/static-assets/html/header.html</header>
-    <footer xsi:nil="false">/static-assets/html/footer.html</footer>
-    <css-group xsi:nil="true">
-        <css>/static-assets/css/style.css</css>
-    </css-group>
-    <!--script-group xsi:nil="true">
-      <script>/static-assets/scripts/prototype.js</script>
-      <script>/static-assets/scripts/crafter-scripts.js</script>
-    </script-group-->
-    <content-type>/acme-com/component/level-descriptor</content-type>
+<levelDescriptor xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <header xsi:nil="false">/static-assets/html/header.html</header>
+  <footer xsi:nil="false">/static-assets/html/footer.html</footer>
+  <css-group xsi:nil="true">
+    <css>/static-assets/css/style.css</css>
+  </css-group>
+  <content-type>/acme-com/component/level-descriptor</content-type>
```

Comment:
Did we really have ACME? Can we fix this? Something like `craftercms/...`

Reply:
We did indeed 🙂 
Let's discuss: I don't see this file in develop but it comes from way back in 2015. There are two more xml files at this level not on develop that somehow are still on 2019.